Students/Jacoby/Mechanum/commands/DriveTrainCommand.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DriveTrain;
+
+/** An example command that uses an example subsystem. */
+public class DriveTrainCommand extends CommandBase 
+{
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final DriveTrain m_subsystem;
+
+
+
+  public DriveTrainCommand(DriveTrain subsystem) {
+    m_subsystem = subsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(m_subsystem);
+  }
+
+
+  @Override
+  public void execute() 
+  {
+    DriveTrain.driveMecanum();
```

Comment:
This method dosent do anything other than read the joystick input. See my other comment in the Drive Train subsystem.